lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -15,7 +15,7 @@ include_once 'dosomething_user.theme.inc';
  */
 function dosomething_user_preprocess_page(&$vars) {
   $school_api_endpoint = variable_get('dosomething_user_school_api_endpoint', 'http://lofischools.herokuapp.com/search');
-  $date_format = (dosomething_settings_get_geo_country_code() === 'US') ? 'MM/DD/YYYY' : 'DD/MM/YYYY';
```

Comment:
Hmm why do we not need this logic anymore? will the POEditor know to translate `MM/DD/YYYY` to the correct string for all users except US?